Resources/views/Section/list.html.twig

```diff
@@ -76,25 +76,29 @@
                         {{ form_start(deleteForm, {"action": path("admin_sectiondelete", {"sectionId": section.id, "redirectErrorsTo": "list"})}) }}
                             {{ form_widget(deleteForm.sectionId) }}
 
-                                <button
-                                    type="button"
-                                    data-universaldiscovery-title="{{ 'section.assign.universaldiscovery.title'|trans({'%sectionName%': section.name })|e('html_attr') }}"
-                                    data-section-rest-id="{{ path( 'ezpublish_rest_loadSection', {'sectionId': section.id}) }}"
-                                    data-section-name="{{ section.name }}"
-                                    class="ez-button ez-button-secondary"
-                                    {% if not canAssign %}disabled="disabled"{% endif %}>{{ 'section.assign.contents'|trans }}</button>
+                            {% set dialogId = 'confirm-section-removal-' ~ section.id %}
+                            {{
+                                include('@EzSystemsHybridPlatformUi/components/confirm_dialog.html.twig', {
+                                    'dialogId': dialogId,
+                                    'message': 'confirm.remove.section'|trans()|desc('Are you sure you want to delete this section?'),
+                                    'confirmButton': deleteForm.delete,
+                                })
+                            }}
 
-                                <a href="{{ path('admin_sectionedit', {'sectionId': section.id}) }}" class="pure-button ez-button ez-button-secondary" data-icon="&#xe606;"{% if not canEdit %} data-disabled{% endif %}>{{ 'section.edit'|trans }}</a>
+                            <button
+                                type="button"
+                                data-universaldiscovery-title="{{ 'section.assign.universaldiscovery.title'|trans({'%sectionName%': section.name })|e('html_attr') }}"
+                                data-section-rest-id="{{ path( 'ezpublish_rest_loadSection', {'sectionId': section.id}) }}"
+                                data-section-name="{{ section.name }}"
+                                class="ez-button ez-button-secondary"
```

Comment:
I think that maybe this is connected to [previous comment on ezp-27573_confirm_mechanism](https://github.com/ezsystems/hybrid-platform-ui/pull/85/files/5560f1d9dbb8bd23a198fdc8890bc92dd2c8fd59#r128779430) but not totally sure